igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/TravisBuildMonitor.java

```diff
@@ -68,7 +68,7 @@
   private final TravisProperties travisProperties;
   private final Optional<EchoService> echoService;
 
-  public static final int TRACKING_TTL = (int) TimeUnit.HOURS.toMinutes(5);
+  public static final int TRACKING_TTL = (int) TimeUnit.HOURS.toSeconds(5);
```

Comment:
Thank you!

Looks like this could be package private / only used in TravisBuildMonitorSpec.  You up to rename this to
```suggestion
  static final int TRACKING_TTL_SECS = (int) TimeUnit.HOURS.toSeconds(5);
```
and maybe even adding secs to the args for BuildCache.setTracking + setTtl so it's more obvious?

Reply:
Done 👍